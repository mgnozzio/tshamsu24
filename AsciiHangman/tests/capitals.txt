CAPITALS
c
a
p
i
t
a
l
s
