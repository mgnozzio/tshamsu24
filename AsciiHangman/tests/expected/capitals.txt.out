Welcome to the ASCII Version of Hangman!
For best results, please run this from the command line.
Please enter a secret word: 
















































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































   ____  
  |    | 
  |      
  |      
  |      
__|__    
Puzzle to solve: _ _ _ _ _ _ _ _ 
Please guess a letter: C
Puzzle to solve: C _ _ _ _ _ _ _ 
Please guess a letter: A
Puzzle to solve: C A _ _ _ A _ _ 
Please guess a letter: P
Puzzle to solve: C A P _ _ A _ _ 
Please guess a letter: I
Puzzle to solve: C A P I _ A _ _ 
Please guess a letter: T
Puzzle to solve: C A P I T A _ _ 
Please guess a letter: A
Puzzle to solve: C A P I T A _ _ 
Please guess a letter: L
Puzzle to solve: C A P I T A L _ 
Please guess a letter: S
Success!  Player 2 wins!
